core/src/com/unciv/logic/city/CityConstructions.kt

```diff
@@ -71,6 +71,8 @@ class CityConstructions {
 
     fun getBasicCultureBuildings() = cityInfo.getRuleset().buildings.values
             .asSequence().filter { it.culture > 0f && !it.isWonder && !it.isNationalWonder && it.replaces == null }
+    
+    fun getCultureBuildings() = getBasicCultureBuildings().map { cityInfo.civInfo.getEquivalentBuilding(it.name) }
```

Comment:
This looks suspicious - `getBasicCultureBuildings` loops all buildings and picks non-uniques, then we go and find the corresponding unique - which again loops all buildings (deferred and early exit nonwithstanding). Also, we translate an object to its String name then back to its Object representation. Lastly, the knowledge only base buildings get into the loop is lost and little later another `getBaseBuilding` is done... Probably doesn't matter much in the big picture. Would matter none if we'd cache the Culture buildings in some lazy on CivilizationInfo or Nation?